feat(content): A Dead Man's Path again (#8654)
endless-sky/endless-sky: data/human/human missions.txt

		fleet "Small Syndicate"
		to spawn
			has "Argosy Hijacking: declined"

mission "A Dead Man's Path 1"
	name "Siblings to New China"
	description "Bring these siblings to New China to hear their father's last will by <date>. They offer to pay you <payment>."
	minor
	passengers 2
	deadline 7
	source
		near "Merak" 3 6
		not attributes "station"
	destination
		planet "New China"
	to offer
		random < 20
	on offer
		conversation
			`The spaceport on <origin> is lively tonight. There are porters running to and from depots with crates, crewmen buying rounds of drinks for each other around the bars before they hit the bed, and bands of Navy officers jovially greeting captains and systematically inspecting ships. The energy and self-purpose of this spaceport is on full display this one night.`

> 			`	The only exception to the bustle and activity are a young woman and man, both dressed very formally, who are seemingly lost and having no luck approaching anybody. You watch they drift from one person to another, each time being sent off without an answer. Finally, they come towards you.`

Reviewer: ```suggestion
			`	The only exception to the bustle and activity are a young woman and man, both dressed very formally, who are seemingly lost and having no luck approaching anybody. You watch them drift from one person to another, each time being sent off without an answer. Finally, they come towards you.`
```

			`	Their faces show clearly that they are on their last legs of hope. It is only with reluctance that the woman says, "Excuse me, are you a spaceship captain?"`
			choice
				`	"Yes, my name's Captain <last>."`
				`	"Depends on what you want."`
					goto suspicious

> 			`	Her face brightens up with a smile, and she continues, "That's good to hear! My brother and I were searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could bring us there? We will pay you for your service, of course."`

Reviewer: ```suggestion
			`	Her face brightens up with a smile, and she continues, "That's good to hear! My brother and I were searching for someone who could give us a ride to <destination>. We received notice that our father's last will is going to be published there. Do you think you could take us there? We will pay you for your service, of course."`
```

				goto helpornot
			label suspicious

> 			`	She laughs. "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could bring us there? We will pay you for your service, of course.`

Reviewer: ```suggestion
			`	She laughs. "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could take us there? We will pay you for your service, of course.`
```

> 			`	She laughs. "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could take us there? We will pay you for your service, of course.`

Reviewer 2: ```suggestion
			`	She laughs. "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to <destination>. We received notice that our father's last will is going to be published there. Do you think you could take us there? We will pay you for your service, of course.`
```
The other one has this.

			label helpornot
			choice
				`	"Of course I can."`
					goto agreed
				`	"I'm sorry, but I've got more urgent things to do. You'll need to find someone else."`

> 			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, with both heads looking down.`

Reviewer 2: > with both heads looking down

Not a huge fan of this.

Author: Why not? The next best option I can think of is:
```suggestion
			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, their heads down.`
```
but it seems like you'd like to avoid that sort of expression?

Reviewer 2: Yeah. It's probably fine though.

				decline
			label agreed
			`	"Fantastic! My name is Sophie, and this is my younger brother," she says, pointing at the man to her side, who goes to shake your hand. "Hi, I'm Liam. We've been looking for a ride for some time now. We were starting to worry that we wouldn't get there in time! You'll have to bring us there quickly."`
			choice
				`	"Don't worry, you'll be surprised how fast my ship flies. We'll get there in no time."`
					goto notime
				`	"Yes, a lot of captains are quite busy these days."`
			`	"It seems that way," Sophie agrees. "Everyone feels more tense or uncertain, and there's always more to do and less time to do it."`
				goto ship
			label notime
			`	Liam's voice barely hides his excitement at finding an available captain. "Well, let's see that ship of yours then, and show us if it's as fast as you say it is!"`
			label ship
			`	You lead them to your ship, and help them with their bags as they board.`
				accept
	on visit
		dialog phrase "generic passenger on visit"
	on complete
		log `Helped a pair of siblings named Sophie and Liam by bringing them to New China so they can hear their father's last will.`

> 		dialog `When you arrive at New China's spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`

Reviewer: ```suggestion
		dialog `When you arrive at <destination>'s spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
```

Author: ```suggestion
		dialog `When you arrive at <planet>'s spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
```
`<planet>` is better than `<destination>` in this instance.

	waypoint "Aspidiske"
	waypoint "Gorvi"
	waypoint "Enif"
	to offer
		has "A Dead Man's Path 1: done"
	on offer
		conversation
			`After waiting two hours in the spaceport's bar, Liam and Sophie come back with unexpectedly broad smiles on their faces. You flag them down from your table and they eagerly take a seat.`
			`	Liam speaks as soon as he can. "Captain <last>, it turns out we will need a bit more than just a ride home! Have you decided to help us again?"`
			choice
				`	"But what do you want me to do?"`
				`	"Sorry, but I'd be stupid to accept a mission I don't know anything about."`
			`	Sophie smiles at her brother. "Of course, Liam just can't wait to get started!" She turns back to you. "Maybe I should start with who our father was. We didn't get to know him very well - he came home only occasionally. Don't get me wrong, our parents weren't fighting, but he had this passion that couldn't make him stay in one place for very long. He was a captain, not unlike you, and he was restless to explore even the furthest parts of the galaxy. We believe he traveled further than most human captains; sometimes he brought small items back home that nobody could explain."`

> 			`	She pauses for a moment. "Anyway, in his testament he told us that he left something behind for us on <planet>, in <system>. He said that he would not make it easy, though; he loved fantasy books, and in those stories there is always something mysterious at play that we do not understand the meaning of until the very end. So he told us of a few systems that we should visit before we go to Prime.`

Reviewer 2: I don't think this explains those broad smiles.

Author: I wrote it under the impression that they would be very excited at the chance of a trip around the entirety of human space in addition to fulfiling their father's wishes and being able to experience his life in some way. After all they expected to be sadder and now they have some sort of mystery on their hands.

			choice
				`	"Which systems would we have to fly through?"`
					goto starmap
				`	"I'm sorry, but that sounds too complicated to me."`
					goto nothanks

			label starmap
			`	Liam pulls a starmap out of his bag. "These are the systems our father wants us to visit: <waypoints>. Those systems are spread far out around human space, but apparently they all have something unique to them. We've never really travelled beyond our home planet, so it'll be quite a big adventure for us! Does that sound good to you?"`
			choice
				`	"Sounds good!"`
				`	"Sorry, I won't be able to take you. Those systems are too far away from each other."`
					goto nothanks
			`	"Great! We weren't expecting this trip, so we'll need to buy some supplies," Sophie says, "but as soon as you're ready we'll be on board."`
				accept

			label nothanks
			`	"Well, it looks like we'll need to find somebody else," Sophie says. They both thank you for bringing them this far and walk away.`
				decline

	on enter "Terminus"

> 		dialog `It's often thought that there are nothing to these uninhabited systems by the Deep, but Sophie points out that there's a strange anomaly in this system. "When we got to see him, our father used to describe all these unusual sights he saw on his adventures flying through wormholes or nebulae. That anomaly reminds me of one of them."`

Reviewer: ```suggestion
		dialog `It is often thought that there are nothing interesting in these uninhabited systems by the Deep, but Sophie points out that there is a strange anomaly in this system. "When we got to see him, our father used to describe all these unusual sights he saw on his adventures flying through wormholes or nebulae. That anomaly reminds me of one of them."`
```

			`	"You're right. I hadn't thought of that," Sophie says."`
			`	All three of you walk around the spaceport district for some time. Coming into a giant plaza, you see one of the newest models from Betelgeuse, a Behemoth, on display in the center. The massive freighter with its many turret emplacements towers over even the plaza, an obvious showcase of Prime's leading shipbuilding industry.`
			`	Liam's face brightens as if he just understood something. "I think our father chose this planet because he was always a fan of Betelgeuse ships. Before he went on his adventures he used to be a test pilot for the shipyard."`
			`	Sophie nods. "Right! And once he was allowed to keep one of their experimental models as a gift for being one of their most productive pilots. Betelgeuse wouldn't let him go when he wanted to leave and start exploring the galaxy! But we're running out of time. Shall we go to the mountain now?"`
			choice
				`	"We should take a shuttle. It'll be the fastest way."`
					goto shuttle
				`	"Let's take the maglev, we should make it before it gets dark."`
					goto maglev
			label shuttle

> 			`	After a short shuttlecraft flight on Prime's extensive and well maintained shuttle service, you and the siblings arrive at a provincial city, where you can see a small mountain that stands alone against the rest of the landscape.`

Reviewer: ```suggestion
			`	After a short shuttlecraft flight on Prime's extensive and well-maintained shuttle service, you and the siblings arrive at a provincial city, where you can see a small mountain that stands alone against the rest of the landscape.`
```

				goto front-mountain
			label maglev

> 			`	You take a maglev from the spaceport and get a tour through some of the regions of Prime's mainland; on the way you pass through the fields of New Toulouse, where the majority of the local agriculture industry is located, and into the New Pyrenees hills. After an hour you and the siblings step off at a provincial city. Although Betelgeuse starts to recede behind the horizon, you can see a small mountain that stands alone against the rest of the landscape.`

Reviewer: ```suggestion
			`	You take a maglev from the spaceport and get a tour through some of the regions of Prime's mainland; on the way you pass through the fields of New Toulouse, where the majority of the local agricultural industry is located, and into the New Pyrenees hills. After an hour you and the siblings step off at a provincial city. Although Betelgeuse starts to recede behind the horizon, you can see a small mountain that stands alone against the rest of the landscape.`
```

			label front-mountain
			`	Liam, with the help of the map, guides all of you to the base of the mountain where you come across a flat wall embedded into a small cavern in its side. He looks a bit embarrassed as he says, "There's a phrase our father left in the instructions: 'Edr- i fen'. I don't know what it means, but I'm certain it has something to do with one of these fantasy novels he used to read."`
			choice
				`	"Sounds funny, but let's say it."`
				`	"You've got a strange dad, but I like his sense of humor."`
			`	Sophie counts to three, and then all of you shout out the phrase together.`

> 			`	Suddenly, a bright blue line appears on the wall. For a second, a flash blinds you and there is a dim hissing noise, like the sound of rock being cut by a laser. As you look back, the line forms into a blue rectangle, the shape rapidly expanding down the wall. When it reaches the ground, the shape has become large enough to cover a human.`

Author: The hissing noise is weak, although I could change it to something else.

			`			"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
			`			"But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
			`			"This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
			`			"Your father."`
			`	Understanding the personal nature of the message, you leave the two of them alone for a bit, walking around and looking at the artifacts on the left of the dome.`

> 			`	After some time, the siblings call you back. Sophie says, "Captain <last>, Liam and I have thought through about this, and we think that we would never be able to live up to our father's expectations if we kept this sword. We can study all these artifacts and understand more about the universe through them, but we cannot study this one. To understand its power, it needs to be used.`

Reviewer: I don't understand why they think the sword is different from the other artifacts. Perhaps some kind of explanation is needed?

> 			`	After some time, the siblings call you back. Sophie says, "Captain <last>, Liam and I have thought through about this, and we think that we would never be able to live up to our father's expectations if we kept this sword. We can study all these artifacts and understand more about the universe through them, but we cannot study this one. To understand its power, it needs to be used.`

Author: I will think of a way to improve this paragraph.

> 			`	After some time, the siblings call you back. Sophie says, "Captain <last>, Liam and I have thought through about this, and we think that we would never be able to live up to our father's expectations if we kept this sword. We can study all these artifacts and understand more about the universe through them, but we cannot study this one. To understand its power, it needs to be used.`

Author: Okay, in the original PR there was a picture of the sword inserted in its pedestal, quite prominent over everything else. I think the reason I rewrote it this way was because the siblings believed that:
a. since their father had this sword by their side during his explorations, it should continued to be used or else "it would be wasted" on them
b. they couldn't take and study everything, given their limited resources

The thing about `a` is that the dialogue doesn't actually mention if the siblings know this, just that they say the sword has been. It is a narrative gap that I haven't thought about yet. A better explanation might be needed, but I will reread the whole thing and see if it really is needed as you've been the first to point this out.

> 			`	After some time, the siblings call you back. Sophie says, "Captain <last>, Liam and I have thought through about this, and we think that we would never be able to live up to our father's expectations if we kept this sword. We can study all these artifacts and understand more about the universe through them, but we cannot study this one. To understand its power, it needs to be used.`

Reviewer 3: I thought that they give you the sword because they don't get into situations where they could ever plausibly use it. If they keep it, it's kind of wasted on them since it'll never get used. This is why they give it to the player, who actually does get into fights and other situations where it could be useful.

> 			`	After some time, the siblings call you back. Sophie says, "Captain <last>, Liam and I have thought through about this, and we think that we would never be able to live up to our father's expectations if we kept this sword. We can study all these artifacts and understand more about the universe through them, but we cannot study this one. To understand its power, it needs to be used.`

Reviewer: Ah, I see! I suppose I just thought there would be more to what the sword does than just chopping things. :-)

Plus, wouldn't they want to put an ancient artifact in a museum rather than have it destroyed through use? But I'm obviously just overthinking things.

> 			`	After some time, the siblings call you back. Sophie says, "Captain <last>, Liam and I have thought through about this, and we think that we would never be able to live up to our father's expectations if we kept this sword. We can study all these artifacts and understand more about the universe through them, but we cannot study this one. To understand its power, it needs to be used.`

Author: > Ah, I see! I suppose I just thought there would be more to what the sword does than just chopping things. :-)

In fact, I did suggest more as well; I had an idea that whenever the sword was swung, an energy bolt like out of an Energy Blaster would be emitted. However, describing that in the dialogue would be clunky, and I don't think it's worth some dialogue and an event to update the sword.

> Plus, wouldn't they want to put an ancient artifact in a museum rather than have it destroyed through use? But I'm obviously just overthinking things.

Emerald Sword.

			`	"This sword was always at our father's side, and it has been many places. Take it to many more." She pulls the sword from the pedestal, and gives it one last look over before handing over it to you. "It is yours now."`
			choice
				`	"I would be more than grateful to accept it."`
					goto accepted
				`	"This is too much for me to take."`
					goto lastchance
			label lastchance
			`	"Are you sure?" Sophie asks. "This is a one time offer. It would be wasted on us."`
			choice
				`	"Yes, I'm sure."`
					goto declined
				`	"Well, if you frame it that way, I'll take it."`
					goto accepted
			
			label accepted
			action
				outfit "Old Man's Sword" 1

> 			`	You take the strange sword from Sophie. It's weirdly light, and you don't really know how to use it yet, but it has some strange feeling of power that seems to emanate from it.`

Reviewer 3: ```suggestion
			`	You take the strange sword from Sophie. It's abnormally light, and you don't really know how to use it yet, but it has some strange feeling of power that seems to emanate from it.`
```

			label conclusion
			`	"Well, we'll need some time to take it all in and arrange things, find a place to stay on Prime and all that," Sophie says, "but we won't keep you from your ship any longer. It was a pleasure to travel with you, Captain <last>."`
			`	Liam adds, "We will always be grateful for you helping us with our father's will. We never imagined that we would come so far and we would have never done it without you. It means so much to us. We wish you well through the stars, Captain <last>."`

			branch accepted-sword extra-money
				not "declined old man's sword"

			label accepted-sword
			action
				payment 50000
			`	After Liam hands you a credit chip for <payment>, you say goodbye to the two siblings and leave them in their father's mysterious dome.`
				decline

			label extra-money
			action
				payment 100000
			`	After Liam hands you a few credit chips for <payment>, you say goodbye to the two siblings and leave them in their father's mysterious dome.`
				decline

			label somewhere-else
			action
				payment 100000

> 			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will. Oh, and here's your payment, and a little extra for putting up with us all this time," Liam says, handing you a few credit chips for <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`

Reviewer 2: > with their heads up

Not a huge fan of this one either.

			label helpornot
			choice
				`	"Of course I can."`
					goto agreed
				`	"I'm sorry, but I've got more urgent things to do. You'll need to find someone else."`
			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport with their heads down.`
				decline
			label agreed
			`	"Fantastic! My name is Sophie, and this is my younger brother," she says, pointing at the man to her side, who goes to shake your hand. "Hi, I'm Liam. We've been looking for a ride for some time now. We were starting to worry that we wouldn't get there in time! You'll have to bring us there quickly."`
			choice
				`	"Don't worry, you'll be surprised how fast my ship flies. We'll get there in no time."`
					goto notime
				`	"Yes, a lot of captains are quite busy these days."`
			`	"It seems that way," Sophie agrees. "Everyone feels more tense or uncertain, and there's always more to do and less time to do it."`
				goto ship
			label notime

> 			`	Liam's voice barely hides his excitement at finding an available captain. "Well, let's see that ship of yours then, and show us if it's as fast as you say it is!"`

Author: I prefer it without the split, it makes him sound more excited to me.

			choice
				`	"Which systems would we have to fly through?"`
					goto starmap
				`	"I'm sorry, but that sounds too complicated to me."`
					goto nothanks

			label starmap
			`	Liam pulls a starmap out of his bag. "These are the systems our father wants us to visit: <waypoints>. Those systems are spread far out around human space, but apparently they all have something unique to them. We've never really travelled beyond our home planet, so it'll be quite a big adventure for us! Does that sound good to you?"`
			choice
				`	"Sounds good!"`
				`	"Sorry, I won't be able to take you. Those systems are too far away from each other."`
					goto nothanks
			`	"Great! We weren't expecting this trip, so we'll need to buy some supplies," Sophie says, "but as soon as you're ready we'll be on board."`
				accept

			label nothanks
			`	"Well, it looks like we'll need to find somebody else," Sophie says. They both thank you for bringing them this far and walk away.`
				decline

	on enter "Terminus"

> 		dialog `It is often thought that there are nothing interesting in these uninhabited systems by the Deep, but Sophie points out that there is a strange anomaly in this system. "When we got to see him, our father used to describe all these unusual sights he saw on his adventures flying through wormholes or nebulae. That anomaly reminds me of one of them."`

Author: I'll make the first suggestion, but those systems where Terminus is are really by the region of the Deep, aren't they?

> 		dialog `It is often thought that there are nothing interesting in these uninhabited systems by the Deep, but Sophie points out that there is a strange anomaly in this system. "When we got to see him, our father used to describe all these unusual sights he saw on his adventures flying through wormholes or nebulae. That anomaly reminds me of one of them."`

Author: ```suggestion
		dialog `It is often thought that there's nothing interesting in these uninhabited systems by the Deep, but Sophie points out that there is a strange anomaly in this system. "When we got to see him, our father used to describe all these unusual sights he saw on his adventures flying through wormholes or nebulae. That anomaly reminds me of one of them."`
```

	on enter "Aspidiske"

> 		dialog `While Sophie and Liam are both quite chatty at times, the beauty of this star stops all their conversations.`

Author: How about this?
```suggestion
		dialog `While Sophie and Liam are both quite chatty at times, the beauty of this star silences them with its awe.`
```

	on enter "Gorvi"
		dialog `Approaching the nebula surrounding Gorvi, Sophie and Liam both chatter to each other about the marvels and mysteries of the Wolf-Rayet star at its center.`
	on enter "Enif"
		dialog `If there was ever a prime example of how much the human race still has to learn, it is the strange, impressive, and yet still unfinished ringworld in this system. Liam tells you, "Our father was a very curious man, and he always talked about the Quarg and their ringworlds, and how it would be another thousand years before humanity could build anything similar. We've never seen how big it was until now!"`

	on visit
		dialog phrase "generic missing waypoint or passengers"

	on complete
		log `Brought Sophie and Liam around the galaxy to various locations that their departed father asked them to visit in his will, before finally bringing them to Prime where he left something for them.`
		conversation

> 			`The siblings are uncharacteristically quiet, perhaps in anticipation of finally arriving at their destination. It is only when you have all stepped out into Prime's spaceport that Sophie says, "Captain <last>, you've helped us immensely, and we can't thank you enough for bringing us all across the galaxy! But we have one last request. Both of us think that you should come with us to see what our father left behind. Do you want to see it?"`

Author: "finally arriving at their final destination" feels tautological.

> 			`The siblings are uncharacteristically quiet, perhaps in anticipation of finally arriving at their destination. It is only when you have all stepped out into Prime's spaceport that Sophie says, "Captain <last>, you've helped us immensely, and we can't thank you enough for bringing us all across the galaxy! But we have one last request. Both of us think that you should come with us to see what our father left behind. Do you want to see it?"`

Reviewer 4: Oops, good catch! Meant this:
```suggestion
			`The siblings are uncharacteristically quiet, perhaps in anticipation of arriving at their destination. It is only when you have all stepped out into Prime's spaceport that Sophie says, "Captain <last>, you've helped us immensely, and we can't thank you enough for bringing us all across the galaxy! But we have one last request. Both of us think that you should come with us to see what our father left behind. Do you want to see it?"`
```

			choice
				`	"Now that we've come so far, sure."`
					goto continue
				`	"I'm sorry, I've got to be somewhere else."`
					goto somewhere-else
			label continue
			`	Liam pulls out a map; not the starmap he had before but a local map of Prime marked 'New Toulouse'. "Apparently our father bought a whole mountain on this planet, and he wants us to go there. It's to the west of us near the New Pyrenees, a little while away from Prime Central Spaceport where we are."`
			choice
				`	"Sounds good. I'll follow your lead."`
					goto shuttle
				`	"We could also have a look around here, and see why your father chose this planet."`
			`	"You're right. I hadn't thought of that," Sophie says."`

> 			`	All three of you walk around the spaceport district for some time. Coming into a giant plaza, you see one of the newest models from Betelgeuse, a Behemoth, on display in the center. The massive freighter with its many turret emplacements towers over even the plaza, an obvious showcase of Prime's leading shipbuilding industry.`

Author: Hmm, this doesn't flow as well to me. Not wrong to me really, but not as nice sounding.

			label enter
			`	In single file, the three of you enter the hallway behind the opening. Strangely, the stone walls are perfectly smoothed off, and the light that guides you through seems to come out of nowhere. "To use alien technology for something like this," Sophie laughs, "that's totally our dad."`
			`	For a long while the three of you continue through the stone hallway, the light following your movements, until it opens up into a dome-shaped room brilliantly bathed in blue light. On the walls, there are all manner of alien artifacts lined up next to each other, but at the center of the room there is a pedestal with a sword inserted into it. A small tablet stands in a socket in front of it.`
			choice
				`	(Go up to the sword.)`
					goto sword-scene
				`	(Take a look around first.)`
			`	You walk around to the right of the dome to the collection of alien objects, all steel in color and rather blocky. "Wow, he must have always brought us back the small items from his expeditions, because these artifacts over here are huge!" Liam exclaims from another part of the dome. "How did he even move all these here?"`

> 			`	Now that Liam points it out, many of the larger artifacts over his way look similar to the Sheragi technology you studied with Albert Foster.`

Author: Yeah, I think it was a little bit of an afterthought in the old PR. I can write a little more, but I don't know enough about the Sheragi to say too much about it (and I'm busy at the moment) so it might take some time.

				to display
					has "Sheragi Archaeology: The Emerald Sword 3: done"
			`	After some time, Sophie calls you both to come and look at the tablet.`
			choice
				`	(Go over to Sophie.)`
			label sword-scene
			`	Approaching the tablet, there is a short message written on it:`
			`			"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
			`			"But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`

> 			`			"This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`

Reviewer 5: Once again, i like commas lol

Also, god damn! This whole tablet part gave me _chills_. I love it!
```suggestion
			`			"This room you are in, now, is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
```

Author: Yeah this doesn't flow as well for me too.
